### ISSUE TITLE
Inconsistent detection of orphaned packages

### ISSUE THREAD

**Issue by xcb-xwii** [lite-xl/lite-xl-plugin-manager#119]
(copied from discord)

when i uninstall a plugin like texcompile, i'm asked to uninstall console, since it became orphaned as it was installed as a dep texcompile 

then i try that w plugins from https://github.com/Evergreen-lxl/evergreen-languages and lpm never asked to uninstall dependencies

really can't figure out why this is so, i don't think there's any errors in the manifest or anything

steps to reproduce:
```sh
lpm install texcompile # also installs console
lpm uninstall texcompile # asks to uninstall console

lpm repo add https://github.com/Evergreen-lxl/evergreen-languages.git:main
lpm install evergreen_cpp # also installs evergreen_c, evergreen
lpm uninstall evergreen_cpp # does not ask anything
```
 
same happens with other plugins in tahat repo such as `evergreen_html_tags`

**Comment by xcb-xwii** [comment]
Seems like the problem is that the addons in `evergreen-languages` all do not specify the `path` field. I tried on a local repo with `path` pointing towards an empty dir and everything then works as expected.

**Comment by adamharrison** [comment]
Should be fixed by f886d77.
